app/dashboard/helpers.py

```diff
@@ -419,6 +420,7 @@ def create_new_bounty(old_bounties, bounty_payload, bounty_details, bounty_id):
                 'bounty_owner_address': bounty_issuer.get('address', ''),
                 'bounty_owner_email': bounty_issuer.get('email', ''),
                 'bounty_owner_name': bounty_issuer.get('name', ''),
+                'bounty_reserved_for': metadata.get('reservedFor', ''),
```

Comment:
You could move `bounty_reserved_for = metadata.get('reservedFor', '')` outside of this `if not` and remove from below and simply grab it once above this if check, so it's already fetched and assigned to be used below.